docs/standard/commandline/snippets/customize-help/csharp/Program.cs

```diff
@@ -0,0 +1,183 @@
+                            "Choose a color that provides enough contrast " +
+                            "with the background color. " + 
+                            "For example, a yellow foreground can't be read " +
+                            "against a light mode background.");
+                })
+                .Build();
+
+        parser.Invoke(args);
+        // </first2columns>
+        Console.WriteLine("First two columns customized.");
+        await parser.InvokeAsync("--light-mode --color Yellow");
+        await parser.InvokeAsync("-h");
+    }
+
+    static async Task DescriptionSection(string[] args)
+    {
+        var fileOption = new Option<FileInfo>(
+            "--file",
+            description: "The file to print out.",
+            getDefaultValue: () => new FileInfo("scl.runtimeconfig.json"));
+        var lightModeOption = new Option<bool>(
+            "--light-mode",
+            description: "Determines whether the background color will be black or white",
+            getDefaultValue: () => true);
+        var foregroundColorOption = new Option<ConsoleColor>(
+            "--color",
+            description: "Specifies the foreground color of console output",
+            getDefaultValue: () => ConsoleColor.White);
+
+        var rootCommand = new RootCommand("Read a file")
+        {
+            fileOption,
+            lightModeOption,
+            foregroundColorOption
+        };
+
+        rootCommand.SetHandler(
+            (FileInfo file, bool lightMode, ConsoleColor color) =>
+            {
+                Console.BackgroundColor = lightMode ? ConsoleColor.Black : ConsoleColor.White;
+                Console.ForegroundColor = color;
+                Console.WriteLine($"--file = {file?.FullName}");
+                Console.WriteLine($"File contents:\n{file?.OpenText().ReadToEnd()}");
+            },
+            fileOption,
+            lightModeOption,
+            foregroundColorOption);
+
+        // <description>
+        var parser = new CommandLineBuilder(rootCommand)
+                .UseDefaults()
+                .UseHelp(ctx =>
+                {
+                    ctx.HelpBuilder.CustomizeSymbol(fileOption,
+                        firstColumnText: "--file <FILEPATH>");
+                    ctx.HelpBuilder.CustomizeSymbol(foregroundColorOption,
+                        secondColumnText: "Specifies the foreground color. " +
+                            "Choose a color that provides enough contrast " +
+                            "with the background color. " +
+                            "For example, a yellow foreground can't be read " +
+                            "against a light mode background.");
+                    ctx.HelpBuilder.CustomizeLayout(
+                        _ =>
+                            HelpBuilder.Default
+                                .GetLayout()
+                                .Skip(1) // Skip the default description section.
```

Comment:
If this is the default _command_ description section, it may help to add that word.